features/hydra/item_uri_template.feature

```diff
@@ -131,8 +131,7 @@ Feature: Exposing a collection of objects should use the specified operation to
     """
 
   Scenario: Get a collection referencing another resource for its IRI
-    When I add "Content-Type" header equal to "application/json"
-    And I send a "GET" request to "/item_referenced_in_collection"
```

Comment:
please revert this change

Reply:
sorry I thought that we wanted to use a json "Accept" on this request, indeed it looks like its a mistake